formats/topical_event_about_page/publisher/links.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "parent"
+  ],
+  "properties": {
+    "parent": {
+      "$ref": "#/definitions/guid_list"
```

Comment:
You can add a `maxItems` property here to enforce that there is only one parent http://spacetelescope.github.io/understanding-json-schema/reference/array.html#length